ontology/yaml/resources/HVAC/entity_types/AHU.yaml

```diff
@@ -4611,4 +4611,164 @@ AHU_DFSS_DX2ZC_ECONZ_CO2C_DSP_RCKTM2X_HTZC_CC2XM_RTM_DTM:
   - RTM
   - DTM
   opt_uses:
-  - condensing_fan_current_sensor
\ No newline at end of file
+  - condensing_fan_current_sensor
+  
+
+
+#PAC Unit #  
+AHU_CHWDC_DFSS_DXSC_DXZTC_EVFSM_REFPM_ZHC_DFRMM_CHWDT:
```

Comment:
Does this handle outside air? If not, its a FCU.

Reply:
This does not handle outside air and these units serves the IDF rooms.